src/Mod/PartDesign/PartDesignTests/TestInvoluteGear.py

```diff
@@ -0,0 +1,118 @@
+#   (c) Jonas Bähr (jonas.baehr@web.de) 2021                  LGPL        *
```

Comment:
Line 1 needs a line of `#**************************` identical to Line 20
Line 2: 
`#    Copyright (c) 2022 Jonas Bähr <jonas.baehr@web.de>                          *`

No need to specify LGPL on line 2 because it's mentioned below

Reply:
It's a copy&paste issue. As far a I see all PD test scripts use this format of the license text. So, it should be fixed in a separate commit.

Reply:
With 6461f9f6d2 I made the license text uniform of the test modules. So can you adjust it in your file too, please?

Reply:
done